schema/rio/v1/crd.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package v1
+
+import "github.com/openfaas/faas-cli/schema"
+
+const APIVersionLatest = "rio.cattle.io/v1"
+
+//RioSpec describe characteristics of the object
+type Spec struct {
```

Comment:
Have you run `gofmt` on this? I like to use VSCode and to install all the `go` tool-chain.

Reply:
I've done this now thanks